wire-ios-request-strategy/Sources/Helpers/FederationTerminationManager.swift

```diff
@@ -79,6 +81,16 @@ private extension FederationTerminationManager {
         }
     }
 
+    func removeConnectedUsers(with domain: String) {
+        let connectedUsersPredicate = ZMUser.predicateForConnectedUsers(hostedOnDomain: domain)
+        let fetchRequest = ZMUser.sortedFetchRequest(with: connectedUsersPredicate)
+        if let pendingUsers = context.fetchOrAssert(request: fetchRequest) as? [ZMUser] {
```

Comment:
using "guard" instead of "if" would make it more readable. Also maybe name should be "connectedUsers" not "pendingUsers"

Reply:
renamed the property